### DISCUSSION TITLE
Device encryption fails

### DISCUSSION THREAD

**Discussion by MisterC925** [pgale/lineage_15.1_Installation_TSV#10] [Q&A]
First I want to thank everyone for their efforts with this box.  It reminds me of the old days when you _had_ to have a CFW on your phone.

I want to use my TSV for work video calls (Google Meet) and to do so I need to install a work profile.

However the work profile requires the device support encryption and throws the error "This device doesn't allow encryption, which is needed for setup..."

The settings show there is hardware backed encryption available, so I tried to encrypt the device.

After using ADB to fake 100% battery it let me proceed with the encryption and momentarily flashed a green wireframe android graphic and then immediately reboots back to the OS.

Is there a way to encrypt the device? (or fool work profile into thinking so?) 

I know this thing runs ancient android and ext4fs but I'm hopeful\

**Comment by pgale** [comment]
I have no idea unfortunately as never needed to do that. Might be best to ask on the discord channel. It's not very active the last couple of weeks though. 

https://discord.gg/D3mgrUbg

Let us know how you get on.
